stdlib/public/core/LifetimeManager.swift

```diff
@@ -265,3 +265,12 @@ extension String {
   }
 }
 #endif
+
+@available(swift, deprecated: 6, message: "Use the copy operator")
```

Comment:
It would be better to rather deprecate this in all language modes, not just in Swift 6+:

```suggestion
@available(*, deprecated, message: "Use the copy operator")
```

Reply:
Fixed